lib/auth/init.go

```diff
@@ -76,6 +77,9 @@ type InitConfig struct {
 	// Backend is auth backend to use
 	Backend backend.Backend
 
+	// ProcessStorage is a backend for local process
+	ProcessStorage *storage.ProcessStorage
```

Comment:
Need to update lib/auth/auth.go:NewServer now to make sure this field is not nil to avoid potential panics.

Reply:
I've added PR with tests fix: https://github.com/gravitational/teleport.e/pull/4594